cluster/sync.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash -e
+
+registry_port=$(./cluster/cli.sh ports registry | tr -d '\r')
+registry=localhost:$registry_port
+
+# Cleanup previously generated manifests
+#rm -rf _out/
+
+# Copy release manifests as a base for generated ones, this should make it possible to upgrade
+#cp -r deploy/converged _out/
+
+# Sed from docker.io to registry:5000
+#sed -i 's/image: docker\.io\/rthallisey\/hyperconverged-cluster-operator:latest/image: registry:5000\/rthallisey\/hyperconverged-cluster-operator:latest/g' _out/operator.yaml
+
+make cluster-clean
```

Comment:
I think if we have make targets running shell scripts it should be a one-way interaction. Either:

- `cluster-clean` should be a dependency on the `cluster-sync` target
- `cluster/sync.sh` calls the appropriate function provided by `cluster/clean.sh` (keeping in mind that I think these should move into the `hack/` directory.